### PR TITLE
AFC-1246 [IMP] Reverted Odoo Waffle and favicon icon to normal.

### PR DESCRIPTION
**Pull request by rupareliyahemangi145**

### CODE REVIEW

File: af_backend_tema/__manifest__.py [hunk 1]
```diff
@@ -4,24 +4,25 @@
...
     "author": "Vertel AB",
     "license": "LGPL-3",
     "installable": True,
     "depends": [
         'base', 'portal', 'web'
     ],
-    'qweb': [
-        'static/src/xml/web.xml',
-    ],
+    # 'qweb': [
```
Comment (MickeHolm) [inline af_backend_tema/__manifest__.py:20]: Why is this commented out and not removed?

File: af_backend_tema/__manifest__.py [hunk 2]
```diff
@@ -4,24 +4,25 @@
...
+    #     'static/src/xml/web.xml',
```
Comment (MickeHolm) [inline af_backend_tema/__manifest__.py:21]: Shouldn't this file be removed as well?
This PR just comment out the reference to the file, but the file is still there.

File: af_backend_tema/__manifest__.py [hunk 3]
```diff
@@ -4,24 +4,25 @@
...
+    # ],
     "data": [
         'views/assets.xml',
         'views/template.xml',
-        'views/webclient_template.xml'
+        # 'views/webclient_template.xml'
```
Comment (MickeHolm) [inline af_backend_tema/__manifest__.py:26]: Shouldn't this file be removed as well?
This PR just comment out the reference to the file, but the file is still there.

**Review by MickeHolm — CHANGES_REQUESTED** [review]

**Comment by rupareliyahemangi145** [comment]
Let me remove files.

**Review by MickeHolm — APPROVED** [review]

**Pull request merged**